I also tried a simple OpenRA bump on openSUSE Tumbleweed and it crashed on startup, when I selected the RA mod it crashed:

```bash
$ result/bin/openra   

(process:3596): Gtk-WARNING **: 00:56:11.910: Locale not supported by C library.
        Using the fallback 'C' locale.
Gtk-Message: 00:56:11.911: Failed to load module "unity-gtk-module"
Gtk-Message: 00:56:11.911: Failed to load module "canberra-gtk-module"
Gtk-Message: 00:56:11.982: GtkDialog mapped without a transient parent. This is discouraged.
exception inside UnhandledException handler: The type initializer for 'System.Console' threw an exception.

[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeInitializationException: The type initializer for 'System.Console' threw an exception. ---> System.TypeInitializationException: The type initializer for 'System.ConsoleDriver' threw an exception. ---> System.Exception: Magic number is wrong: 542
  at System.TermInfoReader.ReadHeader (System.Byte[] buffer, System.Int32& position) [0x00028] in <044e611d6bd84e32a8798850006421d9>:0 
  at System.TermInfoReader..ctor (System.String term, System.String filename) [0x0005f] in <044e611d6bd84e32a8798850006421d9>:0 
  at System.TermInfoDriver..ctor (System.String term) [0x00055] in <044e611d6bd84e32a8798850006421d9>:0 
  at System.ConsoleDriver.CreateTermInfoDriver (System.String term) [0x00000] in <044e611d6bd84e32a8798850006421d9>:0 
  at System.ConsoleDriver..cctor () [0x0004d] in <044e611d6bd84e32a8798850006421d9>:0 
   --- End of inner exception stack trace ---
  at System.Console.SetupStreams (System.Text.Encoding inputEncoding, System.Text.Encoding outputEncoding) [0x00007] in <044e611d6bd84e32a8798850006421d9>:0 
  at System.Console..cctor () [0x0008e] in <044e611d6bd84e32a8798850006421d9>:0 
   --- End of inner exception stack trace ---
  at OpenRA.ExceptionHandler.HandleFatalError (System.Exception ex) [0x00197] in <7923173887f64372a7e3f69359ddd855>:0 
  at OpenRA.Program.<Main>m__0 (System.Object _, System.UnhandledExceptionEventArgs e) [0x00006] in <7923173887f64372a7e3f69359ddd855>:0 
```

. OpenRA installed via my own packages ([stable package](https://build.opensuse.org/package/show/home:fusion809/openra), [bleeding-edge, built from latest git commit](https://build.opensuse.org/package/show/home:fusion809/openra-bleed)) and the official AppImages runs fine, however. Thank you for your efforts to bump this fine game.

--- a/pkgs/games/openra/default.nix
+++ b/pkgs/games/openra/default.nix
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
     owner = "OpenRA";
     repo = "OpenRA";
     rev = "release-${version}";
-    sha256 = "05c6vrmlgzfxgfx1idqmp6czmr079px3n57q5ahnwzqvcl11a2jj";
+    sha256 = "1pgi3zaq9fwwdq6yh19bwxscslqgabjxkvl9bcn1a5agy4bfbqk5";
 
     extraPostFetch = ''
       sed -i 's,curl,curl --insecure,g' $out/thirdparty/{fetch-thirdparty-deps,noget}.sh

